project.md

```diff
@@ -11,41 +11,59 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+I am helping economists (both professionals and students) to be able to perform financial calculations without having to perform them manually, pay for expensive software, use an Google/Excel sheet for just one line, or look for an X formula calculator and use some weird website. With my DSL implementation, their experience could be improved as they could just type one line of code to perform the calculation as they would have in a Google/Excel sheet.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate for the users because it addresses these economists' need to perform calculations without having to open a Google/Excel sheet and have random sheets stored with a few lines of work. Since these users are already having to perform these calculations with some version of typing, whether it be on an online or handheld calculator, they will be able to type a format which follows the conventions of economic formulas and software to input calculations. This means they already have a sufficient technical background with putting variables into a formula so they will not require any more background than is already expected to be able to perform such calculations.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I came up with this idea because I was in my Economics class and I was recalling how bothersome it has been in the past to have to find different calculator tools or open up a Google sheet just for one or two calculations for prior Economics assignments. While I may not have to use these tools in the future, I can appreciate how useful they may be for others in my position who want to just quickly perform a couple calculations for a homework set without having to open a google sheet and then find google sheets documentation on what order the arguments go into the formula. Additionally, I am excited to be able to apply my background in Computer Science to create a tool for people studying another field.
```

Comment:
This Is awesome I love how you were able to come up with an idea based off of problems that you have had in the past.